deploy/grafana-dashboard/queries/environment-mapping.flux

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
where is this line coming from? 😉

Reply:
It bothered me that there was no free line before the environment mapping part and I didn't want to add a line at the bottom of all the other queries :sweat_smile:

Reply:
Ah, I see! What if we modify the `read_query` method to add an empty line there? This would leave the flux files normally and still have the combined result look better.